k-distribution/tests/regression-new/bisimulation/examples/program-equivalence/imp-lambda/main.k

```diff
@@ -0,0 +1,55 @@
+requires "imp-prime.k"
+requires "lambda-prime.k"
+
+module MAIN-SYNTAX
+
+endmodule
+
+module MAIN
+
+imports MAIN-SYNTAX
+
+imports IMP
+imports LAMBDA
+
+imports STRATEGY
+imports RULE-TAG-SYNTAX
+imports K-REFLECTION
+
+configuration <c>
+  <k> $PGM:Pgm </k>
+  <s/>
+  <impT/>
+  <lambdaT/>
+</c>
+
+syntax Bool ::= isSyncPoint(#RuleTag, KItem, K) [function]
+
+rule isSyncPoint(a, K, Ks) => isKResult(K) andBool #isVariable(Ks)
```

Comment:
I understand most of this spec, except the logic around `a` and `b` where are they defined and why are they needed? Perhaps this is worth commenting.